javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,46 @@
+//NOnoN0nOYes (Note taking app)
+
+//Save a note
+
+const notes=[]
+
+function saveNote( content, id){
+    const note = { content, id };
+   notes.push(note);
+}
+saveNote( "Pick up groceries", 1);
+saveNote( "Do laundry", 2);
+
+console.log(notes);
+
+//Get a note
+
+function getNote(id) {
+    const findNote = notes.find((note) => note.id === id);
```

Comment:
Is this variable strictly needed? 😄

Reply:
Do you mean const note? I think I need to declare const note as an object to show that the array notes are an array of objects.  Is not correct?

Reply:
I am referring to `findNote` that is being immediately returned. 😄

Reply:
Oh, yes. The findNote variable in my function searches through an array of notes to find a specific note that matches the given ID. Would you happen to have another suggestion for finding the particular note?